examples/generate.sh

```diff
@@ -83,8 +83,23 @@ else
   exit 1
 fi
 
+# Check if correct controlplane preference is given
 if [[ -n "${3-}" ]] && [[ $3 != -* ]] && [[ $3 != --* ]]; then
-  OUTPUT_DIR=$(echo $3 | tr '[:upper:]' '[:lower:]')
+  if [[ "$3" == "single-node" || "$3" == "multi-node" ]]; then
```

Comment:
not sure if we can have a default value such as single-node so that by default only one machine created?
the new comer might be esaier to get start from our provider instead of learning too much concepts..

Reply:
Sure. Can set the default to "single-node"

Reply:
+1